### ISSUE TITLE
KCD Romandie 2025

### ISSUE THREAD

**Issue by oleg-nenashev** [cncf/kubernetes-community-days#508]
A placeholder for Kubernetes Community Day in 2025. Most likely, it will take place in Lausanne or Geneva, but less common options are possible too. We position it as a local community event, with a mixed-language agenda (English and French). We anticipate having two tracks, and a number of speaking slots dedicated to the local communities and local first-time speakers.

# KCD Organizer Checklist
Please fill in information, organizers, and sponsors as you assemble them, and then make a new comment when this application is ready for review.

### Event Details

* [ ] Event Location: Geneva, CERN Connect (Suisse Romande)
* [ ] Event Date and Time: Winter 2025, tentatively January 28th 2025.
* [ ] Expected Number of Attendees: ~250
* [ ] Event cost per ticket, and if you will have early bird pricing or not: 100...200 CHF

#### Event type:
* [ ] Virtual
* [ ] Hybrid
* [x] In person

#### Organizers Details (identify at least 3 organizers). Designate who will be the CFP admin, as well as the financial officer:

* [ ] Oleg Nenashev, CNCF Ambassador, Gradle, `@Oleg Nenashev`
* [ ] Raphaël Pinson, CNCF Ambassador, Isovalent 
* [ ] Nuno do Carmo, CNCF Ambassador, Suse/Rancher
* [ ] Ricardo Rocha, CNCF TOC, CERN
* [ ] Thierry Predhom, CNC Meetup Organizer, Camptocamp

### Terms

* [x] No other Kubernetes Community Day event in same major city (check with past organizers)
* [x] Minimum of 3 organizers from 3 different organizations
* [x] Every organizer has an account on [Cloud Native Community Groups](https://community.cncf.io/)
* [x] Each organizer lives within the KCD city or just outside of it. For small countries, the organizers can live within the region, but must be able to easily travel to the KCD if it were in-person.
* [x] At least one organizer is a CNCF member employee, CNCF ambassador, or CNCF project maintainer. NOTE: An employee of a member company, qualifies.
* [x] All organizers agree to follow the event guidelines and CNCF Code of Conduct
* [x] All organizer inquiries / communication for CNCF must be summarized via GitHub (comment under your KCD issue) for easy tracking
* [ ] All organizers have taken [Inclusive Open Source Community Orientation (LFC102)](https://training.linuxfoundation.org/training/inclusive-open-source-community-orientation-lfc102/)
* [x] Agree to offer at least 3 complimentary tickets to increase diversity for the event
* [ ] Share the sponsor prospectus with CNCF via your GitHub issue, and link it on your website.
* [x] Agree to ensure the final program schedule is diverse (e.g., not all speakers of one gender, one culture, or from 1 or 2 of the same company). [Here are some tips on how to create a diverse lineup.](https://docs.google.com/presentation/d/1fzT_BdavVKh3mnxxU-PBWyJq9JUfasKwHqekkbYVbw8/edit#slide=id.g56245ab439_0_106)
* [x] Agree to submit your final lineup to kcd@cncf.io for review, prior to publishing it
* [x] Agree to use Cloud Native Community Groups for registration, scheduling, and event website. If using a platform outside of Bevy, organizers must provide CNCF with a full list of attendees.
-- Regardless of your platform choice, please use [these questions](https://docs.google.com/spreadsheets/d/1OEoVQ8Y3eQmEEJeT2abhNOkYIkyk8nAVjebNFvKMTQQ/edit?usp=sharing) to create your registration form
* [ ] Agree to use SM Apply for CFP and to close it at least 6 weeks before the event

### Getting started checklist for organizers

* [ ] Schedule monthly check-ins with KCD Core Organizer(s) (kcd@cncf.io)
* [ ] Minimum of 3 sponsors committed (not required for virtual only events)
CERN - Venue
Sponsor 2 - TBA
Sponsor 3 - TBA
* [ ] Identified fiscal sponsor that can accept payments such as from sponsorships and pay out expenses
Fiscal sponsor that will accept payments
* [ ] Added usernames for [CNCF Slack](https://slack.cncf.io/) for all organizers so that we can add you to the #kcd-organizers channel
* [ ] Create a Twitter handle specific to your KCD event (except for countries like China and Korea where Twitter is not used)

### Getting started checklist for CNCF:
* [ ] Organizer email alias has been created
* [ ] Region created in Bevy
* [ ] Added to Bevy header
* [ ] CFP created in SMapply
* [ ] Marketing Plan shared

### Organizer Planning Timeline
* [ ] Make a copy of [this timeline](https://docs.google.com/spreadsheets/d/1bvCiyyDut1seSnBE6pzVevcJkXLeWbxbncvhFsyY8PI/edit#gid=1736864170) based on whether your event is In-person, Hybrid, or Virtual and link your own spreadsheet here ().
* [ ] To assure you have a diverse lineup, you are required to send a copy of your program to kcd@cncf.io before you publish it.

### Post event checklist for Organizer:
* [ ] Send CNCF event analytics
* [ ] Schedule post mortem meeting
* [ ] Send CNCF videos to be uploaded to YouTube
* [ ] Link videos from YouTube to event page
* [ ] Post event survey sent out

### Post event checklist for CNCF:
* [ ] Removed from Bevy header
* [ ] CFP deactivated
* [ ] Upload videos to YouTube

**Comment by AudMonte01** [comment]
Hi @oleg-nenashev - do you have any other people interested in co-organizing with you? If so, please add them to this issue. Please note that KCD Zurich will occur in the middle of June, so this event will need to happen two months away from it.

**Comment by oleg-nenashev** [comment]
Hello. Yes, planning is in progress. The 2 months gap is not necessary because these are different regions in terms of the language, and there are three hours apart anyway

**Comment by AudMonte01** [comment]
Once the team members are properly added to this issue, schedule a kick-off call with me to allow for this event to proceed with planning. https://calendly.com/audmonte/kcd-onboarding-kickoff-call

**Comment by AudMonte01** [comment]
Hello @oleg-nenashev - Will the majority of the content be in French, or half English?

**Comment by oleg-nenashev** [comment]
50/50, I assume @AudMonte01

**Comment by AudMonte01** [comment]
Here are our only concerns with timing. Let me know your thoughts.
* [KCD Denmark is on November 19](https://kcddenmark.dk/)
* [Cephalocon (LF-hosted) happens December 4-5](https://events.linuxfoundation.org/cephalocon/). Granted it is very niche, focused on one project

**Comment by rochaporto** [comment]
Hi @AudMonte01 . We had a call today and we have a new proposed date: January 28th 2025.

Does this look ok to you?

**Comment by AudMonte01** [comment]
Thank you for the update @rochaporto ~

If you move to 2025, the team will need to wait for the announcements of the new KCD standards, guidelines and benefits, which we will announce at the end of this month.

Regarding the timeline with other events, KCD Oslo typically hosts their event in January. [See their GitHub issue here](https://github.com/cncf/kubernetes-community-days/issues/571). But if this event is a 2.5-hour flight and mostly in French, thee 5-day difference may not be an issue.

Stay tuned,
Audra

**Comment by rochaporto** [comment]
Thanks @AudMonte01. That's fine, the main item we had was about confirming the venue and it seems we can go ahead with the proposed date - let us know if this is not the case. We can wait for the rest when the new KCD details are published.

**Comment by AudMonte01** [comment]
While are excited to see a KCD in Romandie, please do not confirm anything, sell sponsorships, or sign contracts until our team has met with you, reviewed and  approved this event via a kick-off meeting.

Once the new program updates have been released at the end of the month, we will have a formal review process to approve which KCDs can proceed forward. Thank you for your patience through this time.

**Comment by oleg-nenashev** [comment]
Hi @AudMonte01 . How long do you expect the formal process to be? Last time we were waiting for multiple months due to Kubecon, and this time we would need to have a final decision by the CNCF by mid-August so that we could execute on the planning and budgeting for January 28. 

For the context, we have another events franchise confirmed as a plan B, but of course we would prefer to go with KCD if the approval could be granted quickly. We are ready to be a guinea pig for the new process should it be necessary to hit the timeline.

**Comment by AudMonte01** [comment]
Thank you for your patience, we understand KCD 2.0 has taken quite some time. Originally we were waiting on you to gather a team that was ready. Once your team was ready, a new process for KCDs was in the works. Unfortunately a platform that was going to serve us, did not meet its promise when we tested it in May, so we are equally disappointed in the delay caused by this.

The good news is, we are about to send an invitation out for a KCD organizer WG next week. This is so they can be privy to the changes ahead of us publishing our FAQ. Thanks again for all of your patience.

**Comment by AudMonte01** [comment]
Thank you for your submission - we are now ready to accept applications via the [new official application form](https://www.cncf.io/kcds/application/). The deadline for submissions will be Wednesday September 11 at 11:59pm Pacific Daylight Time. More information and the application can be found via https://www.cncf.io/kcds/application/.

This GitHub issue is now closed. Should your KCD be accepted into the 2025 program, a core CNCF staff member will open a new issue for program tracking and transparency.

**Comment by oleg-nenashev** [comment]
New issue: https://github.com/cncf/kubernetes-community-days/issues/612
